src/NServiceBus.Core/Transports/Msmq/QueuePermissions.cs

```diff
@@ -59,8 +59,6 @@ public static void SetPermissionsForQueue(MessageQueue queue, string account)
         {
 
             queue.SetPermissions(LocalAdministratorsGroupName, MessageQueueAccessRights.FullControl, AccessControlEntryType.Allow);
-            queue.SetPermissions(LocalEveryoneGroupName, MessageQueueAccessRights.WriteMessage, AccessControlEntryType.Allow);
```

Comment:
When we create queues, the permissions would be set for Windows Server 2003 and later indirectly by MSMQ. We can leave those lines in without any real benefit as they are already present. If we leave them in I propose a simple comment stating the platforms that would already have these permissions.